### PR TITLE
coalang-facility: Allow external coalang

### PR DESCRIPTION
**Pull request by abhsag24**
Fixes https://github.com/coala-analyzer/coala/issues/2199

@Makman2 does this suffice? @Makman2 @fneu review needed :)

### CODE REVIEW

File: coalib/bearlib/languages/documentation/DocstyleDefinition.py [hunk 1]
```diff
@@ -130,9 +131,13 @@ def load(cls, language: str, docstyle: str):
...
         language_config_parser = ConfParser(remove_empty_iter_elements=False)
+        if not coalang_path:
+            filename = os.path.dirname(__file__) + "/" + docstyle + ".coalang"
+        else:
+            filename = coalang_path
         try:
-            docstyle_settings = language_config_parser.parse(
-                os.path.dirname(__file__) + "/" + docstyle + ".coalang")
+            docstyle_settings = language_config_parser.parse(filename)
+
```
Comment (sils1297) [inline coalib/bearlib/languages/documentation/DocstyleDefinition.py]: unneeded newline, I'm going to remove it and pull it into an own PR ok?

**Comment by sils1297** [comment]
moved to https://github.com/coala-analyzer/coala/pull/2295

File: coalib/bearlib/languages/documentation/DocstyleDefinition.py (hunk 1 above)
Comment (abhsag24) [inline coalib/bearlib/languages/documentation/DocstyleDefinition.py]: yeah sure!
